app/controllers/admin/spam_controller.rb

```diff
@@ -11,7 +10,11 @@ def index
                   else
                     { reviewed: false, approved: false }
                   end
-    @works = ModeratedWork.where(conditions).order(:created_at).page(params[:page])
+    @works = ModeratedWork.where(conditions)
+      .joins(:work)
+      .select("moderated_works.*", "title", "revised_at")
```

Comment:
For this to work you should remove the delegated `title` and `revised_at` functions in `ModeratedWork`.

But it occurred to me that, in addition to the title and date, there's also the list of users associated with the work, which forces the work to be loaded anyway. That could also be made more efficient with a join like `joins(work: :users).group(:id).select("moderated_works.*", "title", "revised_at", "JSON_ARRAY(login) AS json_logins")`, but that would require modifying the `admin_user_links` function, and IDK if the extra complexity is worth the time savings vs. just calling `includes(work: :users)`, which is much more straightforward to understand.